pkg/apis/policies/v1/policy.go

```diff
@@ -98,4 +103,6 @@ type Policy interface {
 	GetUniqueName() string
 	GetContextAwareResources() []ContextAwareResource
 	GetBackgroundAudit() bool
+	GetSeverity() (string, bool)
+	GetCategory() (string, bool)
```

Comment:
Does this enrich the output of `kubectl get [type]policy` by adding a dedicated column?

Reply:
Done in https://github.com/kubewarden/kubewarden-controller/pull/462/commits/423cefd71723bf3606737586d0cfd2a79cb0c301